cms/djangoapps/export_course_metadata/apps.py

```diff
@@ -0,0 +1,18 @@
+"""
+Define the export_course_metadata Django App.
+"""
+
+from django.apps import AppConfig
+
+
+class ExportCourseMetadataConfig(AppConfig):
+    """
+    App for exporting a subset of course metadata
+    """
+    name = u'cms.djangoapps.export_course_metadata'
```

Comment:
New code for python3 doesn't need unicode prefixes. (Here and elsewhere in the PR)